README.md

```diff
@@ -96,8 +96,8 @@ Please see the [Quality Declaration](./QUALITY_DECLARATION.md) for details of th
 
 |CMake project/target   | Current Level | Target Level QNX  | Target Level <br> Linux, Windows, macOS | Comment                             |
 |-----------------------|:-------------:|:-----------------:|:---------------------------------------:|:-----------------------------------:|
-| iceoryx_hoofs         | 2             | 1+                | 1                                       |                                     |
-| iceoryx_posh          | 2             | 1+, 2             | 1                                       | Will be split into separate targets |
+| iceoryx_hoofs         | 2             | 1+                | 1                                       | Except code in the namespace `aux`  |
```

Comment:
`aux` was the best idea @elBoberido and I came up with :thinking: It stands for auxiliary and shall differentiate between core vs. addon/auxiliary. Also see `tools/analyse_blacklisted_namespaces_in_binary.sh` further down.